packages/bpk-component-datepicker/src/BpkDatepicker.js

```diff
@@ -124,7 +124,6 @@ class BpkDatepicker extends Component {
         onChange={() => null}
         onOpen={this.onOpen}
         isOpen={this.state.isOpen}
-        readOnly
```

Comment:
This is optional to include - but I think adding this is redundant here as we already apply it within `withOpenEvents`. If we do want to add it here too I'd probably say we also want to pair it with `aria-readonly=false` - but I think we can just remove it?